django/contrib/admin/templates/admin/nav_sidebar.html

```diff
@@ -1,5 +1,6 @@
 {% load i18n %}
 <button class="sticky toggle-nav-sidebar" id="toggle-nav-sidebar" aria-label="{% translate 'Toggle navigation' %}"></button>
 <nav class="sticky" id="nav-sidebar">
+  <input type="text" id="nav-filter" placeholder="Start typing a content name...">
```

Comment:
Some a11y-related suggestions:

```suggestion
  <input type="search" id="nav-filter" placeholder="Start typing a content name..." aria-label="Filter by model name">
```

I'm also not sure "content" is the right word. Does it search on anything other than models (assuming people aren't adding their own stuff in here)?

Reply:
Yes, it searches only on models, but the word "model" is clear only to dev guys. Most of the people (I mean, real users of the admin UI, ex: content editors, marketers) doesn't have a clue what a "model" is.
BTW, it's called "content type" in Django itself (see `django_content_type` table).
In our company we agreed to call it a "container". It's clear both for devs and less tech-savvy guys.
Options are: "start typing a label", "start typing a content type name". Any other suggestions?

Reply:
Yeah, that makes sense. Maybe we shouldn't specify at all? How about this?

```python
<input type="search" id="nav-filter" placeholder="Start typing to filter..." aria-label="Filter navigation items">
```

Reply:
Much better, thanks!

Reply:
I think we also should mark both these strings for translation with {% translate %}

Reply:
Done